firefox-ios/Client/Frontend/SurveySurface/SurveySurfaceViewController.swift

```diff
@@ -74,7 +74,7 @@ class SurveySurfaceViewController: UIViewController, Themeable {
         label.alpha = 0.0
     }
 
-    private lazy var takeSurveyButton: LegacyResizableButton = .build { button in
+    private lazy var takeSurveyButton: ResizableButton = .build { button in
```

Comment:
The changes in this file should not be part of this PR. It will be part of another task. Could you please undo the changes to `SurveySurfaceViewController`? Thanks 🙏